airflow/providers/jdbc/operators/jdbc.py

```diff
@@ -57,16 +57,21 @@ def __init__(
         jdbc_conn_id: str = 'jdbc_default',
         autocommit: bool = False,
         parameters: Optional[Union[Iterable, Mapping]] = None,
+        handler: Callable = fetch_all_handler,
```

Comment:
```suggestion
        handler: Callable[[CursorResult], Optional[List[Tuple]]] = fetch_all_handler,
```

(plus needed imports)

Reply:
I dont think we should type hint here since the fetchall handler must not be sqlalchemy but bdapi2 compliant (`DbApiHook.run` does not use sqlalchemy).

Does the `CursorResult` import comes from `jaydebeapi` ?

Reply:
It should probably be `Callable[[Any], Any]` then? The goal is to make sure the handler accepts one argument, and returns something.

Reply:
Yes, `Callable[[Any], Any]` should be enough, because initially, I thought about:

```
from airflow.typing_compat import Protocol

class Cursor(Protocol):
    description: Optional[Tuple]
    fetchall: Callable[[], List[Tuple]]

handler: Callable[[Optional[List[Tuple]]], Cursor]
```